homeassistant/components/suez_water/sensor.py

```diff
@@ -84,26 +86,49 @@ def icon(self):
 
     def _fetch_data(self):
         """Fetch latest data from Suez."""
-        from pysuez.client import PySuezError
-
         try:
             self.client.update()
-
+            # _state holds the volume of consumed water during previous day
             self._state = self.client.state
-            self._attributes = self.client.attributes
             self.success = True
+            self._available = True
+            self._attributes[
+                'attribution'] = self.client.attributes[
+                    'attribution']
+            self._attributes['thisMonthConsumption'] = {}
+            for item in self.client.attributes['thisMonthConsumption']:
```

Comment:
What are the items?

Reply:
Daily current month consumption.
I can't know the number of items in advance.